daemon/cmd/daemon.go

```diff
@@ -1159,6 +1168,43 @@ func (d *Daemon) bootstrapClusterMesh(nodeMngr *nodemanager.Manager) {
 	bootstrapStats.clusterMeshInit.End(true)
 }
 
+// ReloadOnDeviceChange regenerates device related information and reloads the datapath.
+func (d *Daemon) ReloadOnDeviceChange(devices []string) {
+	log.WithField(logfields.Devices, devices).Infof("Datapath reload initiated due to changed devices.")
```

Comment:
nit: Infof -> Info

Also wondering if this is crucial information to deliver at Info level or not. In `TriggerReloadWithoutCompile()` for instance we have a similar log at the beginning, but we leave that one at Debug level. Given the previous log messages that I commented about, it should already be clear that device changes have been detected, so there is a sign that Cilium is reacting to these changes. Unless we have significant new information to report here (such as success), I'm not sure that we need to log this. I believe that `TriggerReloadWithoutCompile()` will already cause the appropriate logs to be printed out once this regeneration is complete anyway, so maybe we can either drop this to debug or drop the line entirely?

Reply:
Yep I think the "Devices changed" and the logs from datapath reload are enough.